internal/linters/c/cppcheck/cppcheck.go

```diff
@@ -0,0 +1,21 @@
+package cppcheck
+
+import (
+	"github.com/qiniu/reviewbot/internal/linters"
+	"github.com/qiniu/x/xlog"
+)
+
+var linterName = "cppcheck"
+
+func init() {
+	linters.RegisterPullRequestHandler(linterName, cppcheckHandler)
+}
+
+func cppcheckHandler(log *xlog.Logger, a linters.Agent) error {
+
+	if linters.IsEmpty(a.LinterConfig.Args...) {
```

Comment:
recommend to add a readme to introduce it.

Reply:
done